core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -455,19 +454,20 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
    */
   final def repeatOrElse0[B, E2, C](
     schedule: Schedule[A, B],
-    orElse: (E, Option[B]) => IO[E2, C]
+    orElse: (E, Option[B]) => IO[E2, C],
```

Comment:
In addition to `repeatOrElse0`, there are two other methods that delegate to this one: `repeatOrElse`, and `repeat`. Both of these methods should take a `Clock` which defaults to `Clock.Live`.

Reply:
These are now solved by [this commit](https://github.com/scalaz/scalaz-zio/pull/332/commits/5fa247a94df8893f299bc2a415484e90bf9ce81a)